test/index.js

```diff
@@ -1914,6 +1914,172 @@ describe('control', function() {
 
   });
 
+  describe('allPass', function() {
+
+    it('is a binary function', function() {
+      eq(typeof S.allPass, 'function');
+      eq(S.allPass.length, 2);
+    });
+
+    it('type checks its arguments', function() {
+      assert.throws(function() { S.allPass('wrong'); },
+             errorEq(TypeError,
+                     '‘allPass’ requires a value of type Array ' +
+                     'as its first argument; received "wrong"'));
+    });
+
+    it('returns true when given an empty array of predicates', function() {
+      eq(S.allPass([], 'quiessence'), true);
+    });
+
+    it('returns true when all predicates pass', function() {
+      eq(S.allPass([S.test(/q/),
+                    S.test(/u/),
+                    S.test(/i/),
+                    S.test(/c/)],
+                    'quiessence'),
```

Comment:
:triangular_ruler: `'quiessence'` should be nudged one space to the left to align with the `[`. :)